extra-dev/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.0.3.0/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+maintainer: "pierre-yves@strub.nu"
+homepage: "https://github.com/math-comp/analysis"
+bug-reports: "https://github.com/math-comp/analysis/issues"
+dev-repo: "git+https://github.com/math-comp/analysis.git"
+license: "CeCILL-C"
+authors: [
+  "Reynald Affeldt"
+  "Cyril Cohen"
+  "Assia Mahboubi"
+  "Damien Rouhling"
+  "Pierre-Yves Strub"
+]
+build: [
+  [make "INSTMODE=global" "config"]
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+depends: [
+  "coq" { ((>= "8.7" & < "8.12~") | = "dev") }
+  "coq-mathcomp-field"       {(>= "1.11.0" & < "1.12~")}
```

Comment:
but `coq-mathcomp-field.1.11.0` is not in the `released` repo yet? How is this supposed to work?

Reply:
Ah, I see, it's only supposed to be beta-compatible? But is the plan then to move this package to the `released` repo, or how will clashes be avoided?

Reply:
Yet, it is only supposed to be beta-compatible.
The plan is to have the next version in `released`.